src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceMultiSourceConfig.java

```diff
@@ -81,7 +81,8 @@ public SalesforceMultiSourceConfig(String referenceName,
                               @Nullable String whiteList,
                               @Nullable String blackList,
                               @Nullable String sObjectNameField,
-                              @Nullable String securityToken) {
+                              @Nullable String securityToken,
+                              @Nullable String operation) {
```

Comment:
If we are passing operation in the constructor, we must want to use that value somewhere? Either set the value of the field here or pass it to super().

Reply:
added it to super